CMakeLists.txt

```diff
@@ -97,6 +97,10 @@ if (BUILD_FULL OR BUILD_STATIC OR BUILD_SHARED)
 	add_subdirectory (examples)
 	add_subdirectory (benchmarks)
 
+	if (BUILD_BENCHMARKS_KSLOOKUP)
```

Comment:
its ok to include them always. The variability of the build system should be kept as simple as possible.